test/database.ts

```diff
@@ -2728,12 +2728,10 @@ describe('Database', () => {
     });
 
     it('should accept gaxOpts', done => {
-      const options = {
-        timeout: 1000,
-      };
+      const options = {gaxOptions: {timeout: 1000}};
 
       database.request = config => {
-        assert.deepStrictEqual(config.gaxOpts, options);
+        assert.deepStrictEqual(config.gaxOpts, options.gaxOptions);
```

Comment:
Just a small nit. Possible to make this test a little more robust.

```suggestion
        assert.strictEqual(config.gaxOpts, options.gaxOptions);
```